### PR TITLE
Help improvements

### PR DESCRIPTION
**Pull request by ciyer**
* Rename 'Discourse' to 'Forum'
* Add link to Forum in FooterNavbar
* Add Help Menu

image

Work towards #607 

Test deployment on https://sekhar.dev.renku.ch/

### CODE REVIEW

File: src/landing/NavBar.js [hunk 1]
```diff
@@ -223,6 +223,8 @@ class FooterNavbar extends Component {
           </Nav>
           <Nav className="ml-auto">
             <RenkuNavLink to="/help" title="Help" />
+            { /* <ExternalDocsLink url="https://renku.discourse.group" title="Help Forum" /> */ }
```
Comment (lorenzo-cavazzi) [inline src/landing/NavBar.js]: should we remove this comment?

**Comment by rokroskar** [comment]
I know we decided against this, but would it make sense to also make the gitter link lay-person-friendly (i.e. Gitter --> Chat)?

File: src/landing/NavBar.js (hunk 1 above)
Comment (ciyer) [reply src/landing/NavBar.js]: Oops!

**Comment by ciyer** [comment]
I worry that "Chat" alone might not be clear enough. But the majority of the team was for keeping it as "Gitter".

**Comment by rokroskar** [comment]
"Gitter chat"? 😄

**Comment by lorenzo-cavazzi** [comment]
I wouldn't go with both words since it's a "single word" link in the bottom of the page and multiple words may be confusing -- they may look like 2 separate links, plus there is very little space.

I voted for `Chat` so I may be biased on that direction :smile: 
In any case, I would keep either the generic definition for both links (`Forum` + `Chat`) or the specific one (`Concourse` + `Gitter`). I find the current solution a bit confusing.

**Comment by ciyer** [comment]
Yes, you correctly guessed the purpose. We should probably be consistent with that, though. An alternative is to use a icon.
image

**Pull request merged**